tox.ini

```diff
@@ -46,12 +46,13 @@ deps =
     devdeps: git+https://github.com/spacetelescope/gwcs
     devdeps: git+https://github.com/asdf-format/asdf
     devdeps: git+https://github.com/astropy/asdf-astropy
+    devdeps: git+https://github.com/sunpy/mpl-animators
     # Autogenerate oldest dependencies from info in setup.cfg
     oldestdeps: minimum_dependencies
-    figure: matplotlib==3.9.1
-    figure: mpl_animators==1.1.1
-    figure: astropy==6.1.1
-    figure: ndcube==2.2.2
+    figure-!devdeps: matplotlib==3.9.1
+    figure-!devdeps: mpl-animators==1.1.1
+    figure-!devdeps: astropy==6.1.1
+    figure-!devdeps: ndcube==2.2.2
```

Comment:
Should these packages not be pinned for `figure-devdeps` as well?

Reply:
We install all the git versions of them instead, so we can't have both.

Reply:
Oh yes, I see. Well then won't that (potentially) lead to figure differences as the packages change on git?

Reply:
yeah some of the differences could be something irrelevant has changed upstream, or it could be a legitimate API change. Unfortunately there's no way to test the latter without running the risk of the former. fwiw, random hash breakages don't seem to happen that much on astropy/sunpy.